client/state/data-getters/index.js

```diff
@@ -55,6 +55,7 @@ export const requestSiteAlerts = siteId => {
 								: {} ),
 						} ) ),
 						warnings,
+						updates,
```

Comment:
would you mind putting in the explicit mapping here the way it's done with `threats` above? that way if some data comes in that's invalid or in the wrong shape it will fail instead of propagating deeper?

also, I think we need to send a new `updates` property from the API itself too, no?